src/App.jsx

```diff
@@ -1,44 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [activeTabId, setActive] = useState('tab-13');
```

Comment:
The initial state for `activeTabId` is set to `'tab-13'`, which does not exist in the `tabs` array. According to the task description, if `activeTabId` does not match any object in the `tabs` array, the first one should be active. However, it's better to initialize it with a valid tab id to avoid confusion.